docs/src/Nlay_tutorial.md

```diff
@@ -0,0 +1,151 @@
+# N-layer Cylinder Examples
+
+A major functionality of this package is the ability to perform forward modeling of the diffusion equation in arbitrarily layered media. This page will detail how to use the Nlayer code including the packages `fluence_DA_Nlay_cylinder_CW`, `fluence_DA_Nlay_cylinder_TD`, and `fluence_DA_Nlay_cylinder_FD`. There are also other ppackages for Nlayerd cylinders such as `flux_DA_Nlay_cylinder_CW` that can be used in a similar manner.
+
+## 2-Layer Example
+The first example we will walk through is a simple 2-layered media. We will show how to simulate individual fluence values for the steady-state, time-domain, and frequency-domain as well as how to simulate a vector of fluence values at various times or source detector separations.
+
+Below is an image of the 2-layer system we will be modeling, including the optical properties and thicknesses of each layer as well as the radius of the cylinders shown (a). Note that for most scenarios, you will want to consider a sufficiently large radius to approximate a semi-infinate lateral geometry.
```

Comment:
I think saying "Below is a 2-layer media with example optical properties and layer thicknesses for a cylinder radius of `a=20` cm".  One thing we should be cautious about is encouraging a large radius cylinder as that decreases the convergence of the sum. In reality, if you are using SDS less than 5 cm a radius of 10 cm is practically semi-infinite. We should be careful what we want to encourage or make a note about this.

Reply:
Also infinite is misspelled on the last line